plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcOutputTableHandle.java

```diff
@@ -67,24 +63,54 @@ public JdbcOutputTableHandle(
         this.pageSinkIdColumnName = requireNonNull(pageSinkIdColumnName, "pageSinkIdColumnName is null");
     }
 
+    public JdbcOutputTableHandle(
```

Comment:
Non deprecated constructor should be first. And one constructor should call the other to avoid the copy-paste code.

Reply:
@chenjian2664 The 2nd comment about copy-paste isn't addressed. Is it intentional?

Reply:
@ebyhr I missed this part, not intentional.

Reply:
This constructor still has `@JsonCreator` & `@JsonProperty` annotations and a new constructor doesn't have those annotations. That's weird for me.

Reply:
Do you mean we may let the new added one to call the exists constructor?

Reply:
Can the constructor remain solely, shifting JSON annotations to it? 
Unsure how other repositories depend on this constructor.
Or We may need to keep the `catalogName` , `schemaName`, `tableName` fields and add the new constructor based on the exists one. 
what do you think

Reply:
`RemoteTableName` is already JSON serializable so why do we need new deprecated constructor for just JSON serialization purposes?